app/main.py

```diff
@@ -1 +1,11 @@
 # write your code here
+def copy_file(cmd: str) -> None:
+    cmd_list = cmd.split()
+    if len(cmd_list) == 3 and cmd_list[0] == "cp":
+        _, existing_file, new_file = cmd_list
+    if existing_file != new_file:
```

Comment:
You could refer to non-existing variables `existing_file` and `new_file` in case the first if condition is faulty. Increase the indentation of the second if condition so it would be executed only in case the first one is true.